sys-power/suspend/files/suspend-1.0_p20200924-fix-loff_t-for-musl.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/load.c b/load.c
+index 6d1c6ed..adc0af5 100644
+--- a/load.c
++++ b/load.c
+@@ -8,6 +8,7 @@
+  * This file is released under the GPLv2.
+  */
+
++#define _GNU_SOURCE
+ #include "config.h"
```

Comment:
You should be able to do `AC_USE_SYSTEM_EXTENSIONS` in configure.ac instead.

Reply:
Done